docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md

```diff
@@ -19,6 +19,12 @@ The Linux OS and other software components on each database cluster node must me
 
 Depending on the [provider type](../../yba-overview/#provider-configurations) and permissions you grant, you may have to install all of these requirements manually, or YugabyteDB Anywhere (YBA) will install it all automatically.
 
+{{< warning title="Using disk encryption software with YugabyteDB" >}}
+If you are using third party disk encryption software, such as Vormetric or CipherTrust, the disk encryption service must be up and running on the node before starting any YugabyteDB services. If YugabyteDB processes start _before_ the encryption service, restarting an already encrypted node can result in data corruption.
+
+To avoid problems, pause the universe _before_ enabling or disabling the disk encryption service on universe nodes.
```

Comment:
Could be useful to mention how to do both - pausing a universe and disabling the disk encryption service